### PR TITLE
Dependencies

### PR DESCRIPTION
**Pull request by anle751**

### CODE REVIEW

File: application/src/main/java/com/prjxAPP/Application.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package com.prjxAPP;
```
Comment (VolgaDaniluk) [inline application/src/main/java/com/prjxAPP/Application.java:1]: rename packege from "prjxAPP" to "jungeeks"

File: accounts/build.gradle [hunk 2]
```diff
@@ -0,0 +1,7 @@
+dependencies {
```
Comment (VolgaDaniluk) [inline accounts/build.gradle:1]: is it require to add all this dependencies in each module?

**Comment by VolgaDaniluk** [comment]
is src package still needed?

File: application/src/main/java/com/prjxAPP/Application.java (hunk 1 above)
Comment (anle751) [reply application/src/main/java/com/prjxAPP/Application.java:1]: renamed

File: accounts/build.gradle (hunk 2 above)
Comment (anle751) [reply accounts/build.gradle:1]: removed unnecessary

**Comment by anle751** [comment]
h2 added

**Review by VolgaDaniluk — APPROVED** [review]

**Pull request merged**